### PR TITLE
Fix command names to be consistent with the rest of Theia

### PR DESCRIPTION
**Pull request by vince-fugnitto**
Updated command names to be consistent with other commands.

Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>

### CODE REVIEW

File: packages/cpp/src/browser/cpp-commands.ts [hunk 1]
```diff
@@ -44,17 +44,17 @@ export const SHOW_CLANGD_REFERENCES: Command = {
...
 export const DUMP_INCLUSIONS: Command = {
     id: 'clangd.dumpinclusions',
-    label: 'C/C++: Dump file inclusions (Debug)'
+    label: 'C/C++: Dump File Inclusions (Debug)'
 };
 
 export const DUMP_INCLUDED_BY: Command = {
     id: 'clangd.dumpincludedby',
-    label: 'C/C++: Dump files including this file (Debug)'
+    label: 'C/C++: Dump Files Including this File (Debug)'
```
Comment (benoitf) [inline packages/cpp/src/browser/cpp-commands.ts]: why this is not capitalize ?

File: packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export class PluginExtDeployCommandService implements QuickOpenModel {
 
     public static COMMAND: Command = {
         id: 'plugin-ext:deploy-plugin-id',
-        label: 'Plugin: Deploy a plugin\'s id'
+        label: 'Plugin: Deploy Plugin by Id'
```
Comment (benoitf) [inline packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts]: why this change is not being asked to be reviewed by members of plug-ins work ?

Comment (benoitf) [reply packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts]: For now I'm -1 due to all screenshots/tests including the previous name of the command

Comment (vince-fugnitto) [reply packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts]: Hi @benoitf, yes sorry about that I did mean to have someone from plugin review but I also felt the change of a label was minor to be able to be consistent with the rest of theia. Also while working on a different PR I cam across this command and it didn't make immediate sense to me at what its functionality was as we do not deploy a plugin's id but rather a plugin by id. 

I'm not quite sure as to what screenshots/tests you are referring to but if you'd like me to discard this particular change then I can do so.

Comment (marcdumais-work) [reply packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts]: @benoitf 
> For now I'm -1 due to all screenshots/tests including the previous name of the command

I guess by that you mean that the Che documentation that includes screenshots of the current label and it would be confusing to change it without updating your screenshots?

File: packages/cpp/src/browser/cpp-commands.ts (hunk 1 above)
Comment (vince-fugnitto) [reply packages/cpp/src/browser/cpp-commands.ts]: I initially had it this way but after verbal review from @lmcbout, we decided that `stop words` such as (a, in, this, an) would not be capitalized. Example: `Reveal in Files`.

Do you think we should instead capitalize?

Comment (marcdumais-work) [reply packages/cpp/src/browser/cpp-commands.ts]: This seems like something we should be consistent about. I remember a few related PR discussions, but I do not see anything on the [coding guidelines](https://github.com/theia-ide/theia/wiki/Coding-Guidelines) page. Personally I would keep ```this``` lowercase.

File: packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts (hunk 2 above)
Comment (benoitf) [reply packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts]: yes but let's be consistent then
